### PR TITLE
pricing: adds new boleto fields

### PR DESCRIPTION
**Pull request by sdoblinski**
## Contexto
Adicionar novas taxas de cobrança de boleto à página de pricing

## Checklist
- [x] incremento no objeto de pricing

## Issues linkadas
Resolves https://github.com/pagarme/caesar/issues/517

## Screenshots
78602693-4386f180-782d-11ea-94f4-88fcfccdd7a4

## Notas de deploy
O deploy dessa alteração depende de um pacote de alterações coordenado pelo time Caesars, não fazer o deploy avulso dessa tarefa.

## Como testar?
Alterar as taxas de boleto da company (transaction_cost.boleto, transaction_spread.boleto, boletos.transaction_fixed_fee, boletos.transaction_spread_fee) e confirmar a alteração na página de pricing.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a93c3771eb435f08ce560c747669881cad03810
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a93c3771eb435f08ce560c747669881cad03810
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a93c3771eb435f08ce560c747669881cad03810
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1a93c3771eb435f08ce560c747669881cad03810
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5b1dd955e758c851bfabaf865885d842f6b394e4
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5b1dd955e758c851bfabaf865885d842f6b394e4
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5b1dd955e758c851bfabaf865885d842f6b394e4
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5b1dd955e758c851bfabaf865885d842f6b394e4
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/public/locales/pt/translations.json [hunk 1]
```diff
@@ -1326,14 +1326,17 @@
...
+              "boleto": "Boleto Emitido",
               "antifraud_cost": "Antifraude",
               "mdrs": "MDRS",
               "anticipation": "Antecipação",
               "credito_em_conta": "Crédito de conta",
               "doc": "DOC",
               "ted": "TED",
-              "minimum_monthly_payment": "Mínimo pago mensalmente"
+              "minimum_monthly_payment": "Mínimo pago mensalmente",
+              "paid_boleto": "Boleto Pago",
```
Comment (igorantun) [inline packages/pilot/public/locales/pt/translations.json:1337]: Detalhe: Eu escreveria `Boleto pago` para manter o padrão das outras strings (somente primeira letra maiúscula)

File: packages/pilot/public/locales/pt/translations.json [hunk 2]
```diff
@@ -1326,14 +1326,17 @@
...
+              "transactionFees": "Taxas de Transação",
+              "boletos": "Boleto Pago"
```
Comment (igorantun) [inline packages/pilot/public/locales/pt/translations.json:1339]: Detalhe: Eu escreveria `Boleto pago` para manter o padrão das outras strings (somente primeira letra maiúscula)

File: packages/pilot/public/locales/pt/translations.json [hunk 3]
```diff
@@ -1326,14 +1326,17 @@
...
               "antifraud_cost": "Antifraude",
               "mdrs": "MDRS",
               "anticipation": "Antecipação",
               "credito_em_conta": "Crédito de conta",
               "doc": "DOC",
               "ted": "TED",
-              "minimum_monthly_payment": "Mínimo pago mensalmente"
+              "minimum_monthly_payment": "Mínimo pago mensalmente",
+              "paid_boleto": "Boleto Pago",
+              "transactionFees": "Taxas de Transação",
```
Comment (igorantun) [inline packages/pilot/public/locales/pt/translations.json:1338]: Detalhe: Eu escreveria `Taxas de transação` para manter o padrão das outras strings (somente primeira letra maiúscula)

File: packages/pilot/public/locales/pt/translations.json [hunk 4]
```diff
@@ -1326,14 +1326,17 @@
               "psp": "PSP",
               "credit_card": "Cartão de crédito",
               "debit_card": "Cartão de débito",
-              "boleto": "Boleto",
+              "boleto": "Boleto Emitido",
```
Comment (igorantun) [inline packages/pilot/public/locales/pt/translations.json:1329]: Detalhe: Eu escreveria `Boleto emitido` para manter o padrão das outras strings (somente primeira letra maiúscula)

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 06d9e45304e6de6fe28a0c3a1391f85a7ec18a49
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 06d9e45304e6de6fe28a0c3a1391f85a7ec18a49
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 06d9e45304e6de6fe28a0c3a1391f85a7ec18a49
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 06d9e45304e6de6fe28a0c3a1391f85a7ec18a49
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by igorantun** [comment]
@sdoblinski Pra mim está bom! Só falta corrigir o nome do commit, pro GitCop não reclamar.

Aqui seguimos a padronização `[contexto]: [ação] [descrição]`, então ao invés de `adds new fields to pricing page`, viraria algo como: `pricing: add new boleto fields`.

Me avisa se precisar de qualquer ajuda :)

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eb542025de50a83ee8ec4a12d1dd4cde52dfed3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eb542025de50a83ee8ec4a12d1dd4cde52dfed3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eb542025de50a83ee8ec4a12d1dd4cde52dfed3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eb542025de50a83ee8ec4a12d1dd4cde52dfed3
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by igorantun** [comment]
Boa @sdoblinski! Só esperar mais um approve agora :)

**Comment by ThaisRobba** [comment]
No caso da taxa de transações, usando minha conta de teste, aparece faltando o valor de percentual:

image

Acho que está faltando algum tratamento caso o perfil da pessoa não tenha um dado específico. Erro meu mesmo ou é algo que faltou/

**Comment by sdoblinski** [comment]
> No caso da taxa de transações, usando minha conta de teste, aparece faltando o valor de percentual:
> 
> image
> 
> Acho que está faltando algum tratamento caso o perfil da pessoa não tenha um dado específico. Erro meu mesmo ou é algo que faltou?

Deveria aparecer 0% como no exemplo de Boleto emitido e Cartão de crédito.
A unidade do campo está como `percentage`, isso deveria bastar para a formatação? :thinking:

**Comment by ThaisRobba** [comment]
> Deveria aparecer 0% como no exemplo de Boleto emitido e Cartão de crédito.
> A unidade do campo está como `percentage`, isso deveria bastar para a formatação? 
Não tenho certeza :/ @igorantun Tu sabes?

Qualquer coisa da pra adicionar o default no arquivo `"packages/pilot/src/containers/Settings/Company/GeneralInfoTab/Pricing/index.js"`

Linha 76:
```
{unit === 'percentage' && `${ value || '0' }%`}
```

**Pull request merged**